src/test/run-cli-tests

```diff
@@ -41,7 +41,16 @@ mkdir -p "$FAKE_HOME"
 # cram doesn't like seeing the same foo.t basename twice on the same
 # run, so run it once per directory
 FAILED=0
+FAILEDTOOLS=""
 for tool in "$SRCDIR"/cli/*; do
+    if [ x`uname`x = xFreeBSDx ]; then
+    	if [ x"$tool"x = x"$SRCDIR/cli/rbd"x ]; then
```

Comment:
The rbd tool already exists under FreeBSD. It has different set of commands though, so we need something more complicated here to test FreeBSD version. But in the first just skipping it might be ok.